workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -34,18 +35,24 @@
 S3_BUCKET_NAME = get_env_variable("S3_BUCKET_NAME", "data-refinery")
 S3_COMPENDIA_BUCKET_NAME = get_env_variable("S3_COMPENDIA_BUCKET_NAME", "data-refinery-compendia")
 logger = get_and_configure_logger(__name__)
-
-
-def log_state(message):
-    # get ram usage
-    ram = psutil.virtual_memory().percent
-    # get cpu usage
-    cpu = psutil.cpu_percent()
-
-    logger.debug("cpu:%s - ram:%s - %s", cpu, ram, message)
+### DEBUG ###
+logger.setLevel(logging.getLevelName('DEBUG'))
+
+
+def log_state(message, start_time=False):
+    if logger.isEnabledFor(logging.DEBUG):
+        logger.debug("%s: cpu:%s - ram:%s" % (
+            message,
+            psutil.cpu_percent(),
+            psutil.virtual_memory().percent,
+        ))
+        if start_time:
+            logger.debug('Duration: %s' % (time.time() - start_time))
```

Comment:
Can you still return `start_time` in this condition? It makes me nervous seeing the result being used as a time when it sometimes be None.